.github/workflows/autobuild.yml

```
@@ -102,7 +102,8 @@ jobs:
             cmd3_postbuild:         ./.github/autobuild/linux_deb.sh get-artifacts
             run_codeql:             true
 
-          - config_name:            Linux .deb armhf (artifacts)
+          - config_name:            Linux .deb armhf (artifacts, release only)
+            if:                     contains(needs.create_release.outputs.publish_to_release, 'true')
```

Just curious, is there any advantage of using contains() vs. just a string comparison via `==`?

---

Not sure? I think it's what nefarius used.

---

I can't see any other use of `contains()` in this file. Other similar places just use `==`.
```suggestion
            if:                     needs.create_release.outputs.publish_to_release == 'true'
```

---

I agree it should be changed. Just wanted to say that the workflow *did* use lots of `contains()` until b3f9c2c352cc07b17a6c2a3a1fd210e0bac65818 from 2022-03-23... :)

I think `==` is more appropriate here.